ext/travisci/install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Shouldn't need to bring this in either assuming we don't bring in the line from the .travis.yml file that references it.